tests/kafkatest/tests/client/client_compatibility_features_test.py

```diff
@@ -79,14 +79,16 @@ def __init__(self, test_context):
             "replication-factor": 3
             }}
         self.kafka = KafkaService(test_context, num_nodes=3, zk=self.zk, topics=self.topics)
+        # Always use the latest version of org.apache.kafka.tools.ClientCompatibilityTest
+        # so store away the path to the DEV version before we set the Kafka version
+        self.dev_script_path = self.kafka.path.script("kafka-run-class.sh", self.kafka.nodes[0])
 
     def invoke_compatibility_program(self, features):
-        # Run the compatibility test on the first Kafka node.
```

Comment:
nit: no need to remove this comment.  Otherwise LGTM -- thank you again!

Reply:
will copy that.